web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -121,7 +121,34 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
 
         this._loadMenuItemsFromHandlers(processedValues, collection);
 
-        return this._applyTemplate(tpl, processedValues);
+        if (this._spatialSubsetIntersect(collection)) {
+            return this._applyTemplate(tpl, processedValues);
+        } else {
+            return this._emptyMsg();
+        }
+    },
+
+    _spatialSubsetIntersect: function(collection) {
```

Comment:
If I understand the code correctly, then the subsetintersect is only being applied for ncwms. I wonder if the method name should be changed to reflect that? 

Also, why is data being returned that is outside the bound of interest?